core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -1606,6 +1574,25 @@ class DAGSchedulerSuite
     assertDataStructuresEmpty()
   }
 
+  test("accumulators are updated on exception failures") {
```

Comment:
actually, maybe not because in the future if we do expose `countFailedValues` to the user this comment will be outdated and may be hard to find (since there's another one in `TaskContextSuite`). This test is just testing the general functionality that happens to be used only for internal accumulators currently. I think it's OK to leave it out.